squidDrone/src/com/com_interface.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef SRC_COM_INTERFACE_HPP_
+#define SRC_COM_INTERFACE_HPP_
+
+#ifndef UNIT_TEST
+#include "com_message_buffer.hpp"
+#else
+#include "com_message_buffer_mock.hpp"
+#endif
+
+#include <array>
+
+namespace com {
+
+  /**
+   * @class ComInterface 
+   * @brief This is a purely virtual class to act as
+   * a facade for the device specific implmentation.
+   * 
+   */
+  class ComInterface {
+    public:
+      /** 
+      * @brief Default constructor may not be used.
+      * 
+      */
+      ComInterface() = delete;
+
```

Comment:
virtual destructor is missing, but needed.

Reply:
Added destructor